File: backend/app/core/supabase.py
import os
from dotenv import load_dotenv
from supabase import create_client, Client

load_dotenv()

API_URL = os.getenv("SUPABASE_URL")
API_KEY = os.getenv("SUPABASE_KEY")

supabase: Client = create_client(API_URL, API_KEY)

File: frontend/SoliasArt/src/services/supabase.js
import { createClient } from "@supabase/supabase-js";

const supabaseUrl = "https://cyishjpdiwdxnceaiahp.supabase.co";
const supabaseKey = "sb_publishable_9uuxIMAFrGeEAQtfMqAE2Q_PBlduT2G";

export const supabase = createClient(supabaseUrl, supabaseKey);